apps/tx-builder/src/hooks/useServices/interfaceRepository.ts

```diff
@@ -76,14 +75,12 @@ class InterfaceRepository {
       .map((m: any) => {
         return {
           inputs: m.inputs || [],
-          name: m.name || (m.type === 'fallback' ? 'fallback' : 'receive'),
+          name: m.name || (m.type === 'fallback' ? 'fallback' : m.type === 'constructor' ? 'constructor' : 'receive'),
```

Comment:
Could this include a little bit more context why it's needed? Comment or a function with a meaningful name that explains what it does

Reply:
You have the detailed info in #109, but basically the real method is `constructor` (looking at the ABI response from etherscan) instead of `receive`

Reply:
Do you really expect someone searching half of GitHub when stumbling upon this unreadable piece of code? Yes, I do have the info, but someone other reading it won't.

Reply:
Also, you cannot interact with a constructor but it's still shown
From #109 
>The real method would be the constructor (looking at the ABI response from etherscan), which is not interactable on deployed contracts.

Reply:
My understood of the problem was to show the method `constructor` with the parameter `_masterCopy`.  Could you @rmeissner clarify this to me?

Reply:
We should not display the constructor as you cannot call it on a deployed contract. Constructirs are only available when deploying a new contract, but not during runtime as their code is not stored on chain.

We should still properly handle the receive method. E.g. if there is a payable receive or fallback function (or if we have no abi at all) we should display the eth field to generate a tx that only transfers eth. We should never show the constructor.

Reply:
Also I think we should filter by function types we support. Currently we filter by the types we do not support (e.g. event) but if solidity adds a new type it will break the tx builder

Reply:
> we should display the eth field to generate a tx that only transfers eth.

This was already done in my pr and is already merged.

Reply:
Updated!